### PR TITLE
Add ability to regenerate a unique token with the same options

### PR DESCRIPTION
**Pull request by amrrbakry**
Adds the ability to regenerate a unique token with the same options.

Example:

    class Example < ActiveRecord::Base
      has_unique_field :token, length: 12
    end

    example = Example.create
    example.token # 2RCYqmsZ7dYG

    example.regenerate_token # true
    example.token # P2WRSQjhYYkq
    # generated a new unique token with the same length

### CODE REVIEW

File: spec/uniqueness/uniqueness_spec.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@
       it { expect(page.token.length).to eq 12 }
     end
 
-    context 'excludes ambigious characters from human field', focus: true do
+    context 'excludes ambigious characters from human field' do
```
Comment (owahab) [inline spec/uniqueness/uniqueness_spec.rb]: Nice catch

**Comment by owahab** [comment]
Please update Gemfiles.

**Comment by amrrbakry** [comment]
@owahab done :+1:

**Comment by rkhater** [comment]
@amrrbakry good add, Published to ruby gems.

**Comment by amrrbakry** [comment]
@rkhater Thanks! happy to help.

**Pull request merged**